clmm/modbackend/func_layer.py

```diff
@@ -0,0 +1,509 @@
+# Thin functonal layer on top of the class implementation of CLMModeling .
```

Comment:
Is there a reason for this and `generic.py` being separated files?

Reply:
Yes, func_layer.py  is just a thin layer providing a functional interface for the OO CLMModeling object (implemented by an actual back-end). The file generic.py contains all functions that can be used by any back-end.